src/fzx/tui/term_app.hpp

```diff
@@ -41,7 +41,9 @@ struct TermApp
   std::set<uint32_t> mSelection = {};
 
 private:
+  std::string_view mPrompt = ">";
```

Comment:
Since this will probably be configurable at runtime, std::string here
```suggestion
  std::string mPrompt = ">";
```